src/components/join-flow/join-flow-steps.jsx

```diff
@@ -8,8 +8,11 @@ const {injectIntl, intlShape} = require('react-intl');
 
 const validate = require('../../lib/validate');
 const FormikInput = require('../../components/formik-forms/formik-input.jsx');
+const FormikSelect = require('../../components/formik-forms/formik-select.jsx');
 const JoinFlowStep = require('./join-flow-step.jsx');
 
+require('./join-flow-steps.scss');
+
```

Comment:
Can the eslint disable be removed now?

Reply:
I split the files up -- i think it's better and clearer that way. So I was able to remove the eslint disable.